tests/unittest/_torch/test_executor_request_queue.py

```
@@ -316,141 +316,380 @@ def test_clear_canceled_req_ids(executor_queue):
+        attention_dp_queue.waiting_queue, 3, enable_attention_dp=True)
+
+    assert len(result) == 2
+    assert req2 in result
+    assert req3 in result
+    assert req1 not in result
+
+
+def test_can_process_attention_dp_request(attention_dp_queue):
+    req_no_params = RequestQueueItem(1, Mock())
+    assert attention_dp_queue._can_process_attention_dp_request(
+        req_no_params, [0, 0, 0, 0]) == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `...:` for truth checks

Replace comparison

(E712)